stripe-crypto-onramp-example/src/main/java/com/stripe/android/crypto/onramp/example/OnrampViewModel.kt

```diff
@@ -36,16 +38,12 @@ internal class OnrampViewModel(
 
     val onrampCoordinator: OnrampCoordinator
 
-    private val _uiState = MutableStateFlow<OnrampUiState>(OnrampUiState.Loading)
+    private val _uiState = MutableStateFlow(OnrampUiState())
```

Comment:
👀 Refactored by adding a `Screen` enum and keeping all state in a single data class. Should be more DRY and easier to follow now.